core/esp_spi.c

```diff
@@ -221,6 +221,26 @@ uint32_t spi_transfer_32(uint8_t bus, uint32_t data)
     return res;
 }
 
+static void _rearm_extras_bit(uint8_t bus, bool arm) {
+
+    if(!_minimal_pins[bus]) return ;
+    static uint8_t status = 0 ;
```

Comment:
Well, i will take readable option then. We can optimise spi in other way if needed.